src/com/vivian/java/leetcode/Q171_ExcelSheetColumnNumber.java

```diff
@@ -0,0 +1,19 @@
+package com.vivian.java.leetcode;
+
+public class Solution {
```

Comment:
Please comment Clarification, Approach, Edge Case and Complexity.

Reply:
that's your contribution,but I will do it what you commit above